doc/ref/configuration/master.rst

```diff
@@ -2089,10 +2089,21 @@ master, specify the higher level master with this configuration value.
 
     syndic_master: masterofmasters
 
+You can optionally connect a syndic to multiple higher level masters by
+setting the 'syndic_master' value to a list:
+
+.. code-block:: yaml
+
+    syndic_master:
+      - masterofmasters1
+      - masterofmasters2
+
+Each higher level master must be set up in a multimaster configuration.
```

Comment:
Not sure what this means? In the case I us its just N regular Master of masters-- Although IIRC you have to set master_id or something